misc/jhbuildrc-custom

```diff
@@ -6,7 +6,7 @@ import os
 checkoutroot = os.path.expanduser("~/jhbuild_checkoutroot")
 prefix = os.path.expanduser("~/jhbuild_prefix")
 modulesets_dir = os.environ["QL_OSXBUNDLE_MODULESETS_DIR"]
-moduleset = ["bootstrap", "quodlibet"]
```

Comment:
yes, it finds that on its own. plus, the sample `jhbuildrc` doesn't have it: https://git.gnome.org/browse/gtk-osx/tree/jhbuildrc-gtk-osx-custom-example#n38